packages/utils/odsp-doclib-utils/src/odspErrorUtils.ts

```diff
@@ -250,6 +250,10 @@ export function createOdspNetworkError(
 				error = new FluidInvalidSchemaError(errorMessage, driverProps);
 				break;
 			}
+			if (innerMostErrorCode === "fluidEpochNotProvided") {
+				error = new RetryableError(errorMessage, OdspErrorTypes.usageError, driverProps);
```

Comment:
As discussed in standup, make sure someone knowledgeable about odsp/epochTracker comments on whether this is OK to make retryable.

Reply:
After chatting privately with Vlad, we assessed the risk of current approach and explore other solutions like stashing the epoch value. 

Stashing epoch would add extra complexity and won't be useful as it can change at anytime while we load offline so it does not help us with file validation. Given we don't have epoch value at loading from pending state anyway, we are effectively missing all of that validation, but given other mechanisms to accomplish the same while loading, the risk of data corruption is partially mitigated.